### PR TITLE
Fix: Add i18n label for OpenSnsTokenSwap action

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

"Type" shows undefined for OpenSnsTokenSwap actions.

# Changes

* Add i18n label for OpenSnsTokenSwap action

# Tests

Not applicable.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@mstrasinskis @peterpeterparker please review

File: frontend/src/lib/i18n/en.governance.json [hunk 1]
```diff
@@ -78,7 +78,8 @@
     "AddOrRemoveNodeProvider": "Add or Remove Node Provider",
     "SetDefaultFollowees": "Set Default Followees",
     "Motion": "Motion",
-    "SetSnsTokenSwapOpenTimeWindow": "Start decentralization sale"
+    "SetSnsTokenSwapOpenTimeWindow": "Start decentralization sale",
+    "OpenSnsTokenSwap": "Open Decentralized Token Sale"
```
Comment (peterpeterparker) [inline frontend/src/lib/i18n/en.governance.json:82]: nth: one label begins with all capital letters "Open Decent..." and the other with only a capital letter on first word "Start decen...", should it be aligned?

**Review by peterpeterparker — APPROVED** [review]
LGTM

File: frontend/src/lib/i18n/en.governance.json (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/i18n/en.governance.json:82]: Yes, I'll capitalize the previous one. But I think that it's not used anymore. I asked but I'll remove it in another PR if they answer late.

Comment (lmuntaner) [reply frontend/src/lib/i18n/en.governance.json:82]: done

**Pull request merged**